src/hci/gnomex/controller/ChangePassword.java

```diff
@@ -203,34 +203,34 @@ else if (appUser.getIsActive() != null && appUser.getIsActive().equals("N")) {
       this.validate();
     } catch (HibernateException e) {
       e.printStackTrace();
-      log.error(e.getClass().toString() + ": " + e);
+      log.error(e.getClass().toString() + ": ", e);
```

Comment:
Or, since all of the catch blocks do the same thing, just keep catch (Exception e) and remove the rest.